voluncheer/profiles/views/profile.py

```diff
@@ -101,26 +104,76 @@ def password_reset_request(request):
 
 def profile_update(request):
     """Get profile update POST and call save function on ChangeForms."""
-    profile = get_object_or_404(User, pk=request.user.pk)
     if request.user.is_volunteer:
+        profile = get_object_or_404(Volunteer, pk=request.user)
         form = VolunteerChangeForm(
             request.POST,
             request.FILES,
             instance=profile,
         )
     elif request.user.is_organization:
+        profile = get_object_or_404(Organization, pk=request.user)
         form = OrganizationChangeForm(request.POST, request.FILES, instance=profile)
     else:
         raise ValueError("profile_update: user must either a volunteer or an organizaiton.")
     form.save()
     return redirect("profile")
 
 
+"""
+This part is for Volunteer specified features.
```

Comment:
For non-docstring comments use the # format: https://peps.python.org/pep-0008/#block-comments